pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -585,7 +640,61 @@ func (m *kubeGenericRuntimeManager) SyncPod(pod *api.Pod, _ api.PodStatus, podSt
 		return
 	}
 
-	// Step 5: start containers in podContainerChanges.ContainersToStart.
+	// Step 5: start init containers.
+	next, status, done := findActiveInitContainer(pod, podStatus)
+	if status != nil {
```

Comment:
nit: `status != nil && status.ExitCode != 0` saves an indentation and I don't think it's less readable